README.md

```diff
@@ -149,24 +151,84 @@ loadConfig[Conf](conf)
 ```
 
 
-## Customizing naming conventions
+## Extend the library to support new types
+
+Not all types are supported automatically by `pureconfig`. For instance, classes
+that are not case classes are not supported out-of-the-box:
+
+```scala
+> class Foo(var bar: Int) { override def toString: String = s"Foo(${bar.toString})" }
+> loadConfig[Foo]
+could not find implicit value for parameter conv: pureconfig.ConfigConvert[Foo]
```

Comment:
I agree with you but although it appears in the diff, I only moved this chunk of text from one place to another. The README is pretty rich in content, but lacks an uniform writing style - not only in this case but in several others that doesn't appear in the diff.

Do you mind if we leave this to another pull request? I don't mind scanning the README and making the examples more uniform.